pkgs/development/python-modules/json2html/default.nix

```
@@ -0,0 +1,33 @@
+{
+  buildPythonPackage,
+  lib,
+  python3,
+  fetchFromGitHub,
+}:
+
+buildPythonPackage rec {
+  pname = "json2html";
+  version = "1.1.1";
```

```suggestion
  version = "1.3.0";
```
Pypy has 1.3.0.
https://pypi.org/project/json2html/

https://github.com/softvar/json2html/releases/tag/v1.3.0

Why 1.1.1?

---

now using github 1.3.0